main/http_server/forge-os/src/app/layout/styles/theme/theme-base/components/input/_dropdown.scss

```
@@ -63,6 +63,67 @@
+    background: $inputBg;
+    border: $inputBorder;
+    transition: $formElementTransition;
+    border-radius: $borderRadius;
+    color: $inputTextColor;
+
+    &:not(.p-disabled):hover {
+        border-color: $inputHoverBorderColor;
+    }
+
+    &:not(.p-disabled).p-focus {
+        @include focused-input();
```

Stylelint (17.13.0)

[error] Unexpected parentheses in argumentless mixin "focused-input" call (scss/at-mixin-argumentless-call-parentheses)

(scss/at-mixin-argumentless-call-parentheses)